adagrams/game.py

```diff
@@ -1,11 +1,149 @@
+import random
+
 def draw_letters():
-    pass
+    """
+    input: nothing
+    output: 10 random tiles in a list randomly taken from alphabet dictionary.
+    """
+    allletters = []
+    alphabet = {
```

Comment:
Since we wont' be modifying this dictionary, how about making this into a constant? 🤔